liferay-tango/liferay-portal
# LPS-122799 Show warning message when role segmentation is disabled.

**Motivation**

As we are disabling the Assign Roles by segment by default, we have to warn users in the role configuration screen that this kind of assignment won't work.

**Proposed Solution**

Show warning message when role segmentation is disabled

![Screenshot 2020-11-11 at 13 12 02](https://user-images.githubusercontent.com/15845466/98810749-f2bd2400-241f-11eb-930d-ae4dc588a1f4.png)

**How to test it**

1. Go to App Menu > Roles > Create Role > Select Segments
2. The warning message is shown. 
3. Go to System Settings > Segments > Segments Services (and enable Segmentation by roles)
4. The warning message is not shown.
